e2e/core/pages/metadata-view.page.ts

```diff
@@ -107,28 +106,16 @@ export class MetadataViewPage {
         await BrowserVisibility.waitUntilElementIsNotVisible(this.editIcon);
     }
 
-    async editIconClick(): Promise<void> {
-        await BrowserActions.clickExecuteScript('button[data-automation-id="meta-data-card-toggle-edit"]');
-    }
-
-    async informationButtonIsDisplayed(): Promise<void> {
-        await BrowserVisibility.waitUntilElementIsClickable(this.informationButton);
-    }
-
-    async informationButtonIsNotDisplayed(): Promise<void> {
-        await BrowserVisibility.waitUntilElementIsNotVisible(this.informationButton);
+    async generalEditIconDisplayed(): Promise<void> {
+        await BrowserVisibility.waitUntilElementIsVisible(this.editIconGeneral);
     }
 
-    async clickOnInformationButton(): Promise<void> {
-        await BrowserActions.click(this.informationButton);
-    }
-
-    async getInformationButtonText(): Promise<string> {
-        return BrowserActions.getText(this.informationSpan);
+    async editIconClick(): Promise<void> {
+        await BrowserActions.clickExecuteScript('button[data-automation-id="meta-data-card-toggle-edit"]');
     }
 
-    async getInformationIconText(): Promise<string> {
-        return BrowserActions.getText(this.informationIcon);
+    async editIconGeneralClick(): Promise<void> {
```

Comment:
functions names should start with verb, please rename - it should be something like clickEditIconGeneral